pkg/util/errors/errors.go

```diff
@@ -2,8 +2,9 @@ package errors
 
 // Generic Agent Errors
 var (
-	ErrInitServicesNotReady         = New(1001, "failed to initialize. Services are not ready")
-	ErrTimeoutServicesNotReady      = New(1002, "failed with timeout error.  Services are not ready")
-	ErrPeriodicCheck                = Newf(1003, "%s failed.  Services are not ready")
-	ErrStartingPeriodicStatusUpdate = New(1004, "error starting periodic status update")
+	ErrInitServicesNotReady          = New(1001, "failed to initialize. Services are not ready")
+	ErrTimeoutServicesNotReady       = New(1002, "failed with timeout error.  Services are not ready")
+	ErrPeriodicCheck                 = Newf(1003, "%s failed.  Services are not ready")
+	ErrStartingPeriodicStatusUpdate  = New(1004, "error starting periodic status update")
```

Comment:
Maybe even combine them to status update rather than periodic vs immediate?

Reply:
Yup, will update.